diff --git a/src/rollup-declarations.js b/src/rollup-declarations.js
index b86dfbe44eb76a3f923ebd8ad2416edb8cf8b833..b5ae5b53313297b1650b7a6a95d1531974e0dd16 100644
--- a/src/rollup-declarations.js
+++ b/src/rollup-declarations.js
@@ -7,6 +7,7 @@ exports.default = rollupDeclarationsPlugin;
 const execa_1 = __importDefault(require("execa"));
 const walk_sync_1 = __importDefault(require("walk-sync"));
 const promises_1 = require("fs/promises");
+const fs = require("fs");
 function rollupDeclarationsPlugin(declarationsDir) {
     let glintPromise;
     return {
@@ -29,6 +30,9 @@ function rollupDeclarationsPlugin(declarationsDir) {
     };
 }
 async function fixDeclarationsInMatchingFiles(dir) {
+    if (!fs.existsSync(dir)) {
+      return;
+    }
     const dtsFiles = (0, walk_sync_1.default)(dir, {
         globs: ['**/*.d.ts'],
         directories: false,
