diff --git a/lib/git.js b/lib/git.js
index 31de3ccb358a802b2a02c506473dc8cfdd7f36ad..fd445517165375db1afca19fd7268df75a619157 100644
--- a/lib/git.js
+++ b/lib/git.js
@@ -45,6 +45,11 @@ function parseLogMessage(commit) {
 }
 exports.parseLogMessage = parseLogMessage;
 function listCommits(from, to = "") {
+    if (from === 'ember-headless-table@2.1.4') {
+      // first commit after hard-fork
+      from = 'f49c0a0';
+    }
+
     return execa
         .sync("git", [
         "log",
